server/app/services/ambassadors.js

```diff
@@ -127,7 +132,7 @@ async function signup(json) {
     phone: normalize(json.phone),
     email: json.email || null,
     date_of_birth: json.date_of_birth || null,
-    address: JSON.stringify(json.address),
+    address: JSON.stringify(address),
```

Comment:
maybe we can save `address.state.toUpperCase()`?

Reply:
@brodavi I convert `address.state` to uppercase [on line 64](https://github.com/colab-coop/HelloVoter/pull/48/files#diff-6a461ccae6b60fe167e2ab8f3db267c9R64) so it is saving to the database all caps already. I double checked in Neo4j and it looked like it was saving correctly.